katago_server/trainings/models/network.py

```diff
@@ -52,6 +52,9 @@ class Meta:
         ordering = ["-created_at"]
 
     id = BigAutoField(primary_key=True)
+    # TODO is it possible to make this the primary key?
+    # Having a url like https://katago.tycoach.me/api/networks/2/ is not very useful if the intended name for users
+    # to recognize the network is not "2", but rather this name.
```

Comment:
Same as before, look up for "lookup_field" :)

Also is network name really unique ? I think it is only unique inside a run so we should remove unique true and add 
https://stackoverflow.com/a/56024282

Reply:
Network name being unique even across runs is probably a good practice, so I'm happy to have that restriction. If we ever had multiple runs wouldn't be happy with an overlapping name for user-friendliness because then when the users run download and use it, it would be confusing to talk about the model or remember which run it came from.

Reply:
But name for me is the part: `s4384473088-d968438914`.
Along with run name: `g170e` and the network info `b20c256x2`  it makes the full filename: `g170e-b20c256x2-s4384473088-d968438914.zip`

Reply:
When I write the network upload code, name will be like "g170e-b20c256x2-s4384473088-d968438914". Yes this is slightly redundant, but it means "name" can be the single universal identifier for the network. The KataGo code also calls this the "name" of the network when it is logging the network, or creating the network file (the network file internally contains its name, so that it can still be uniquely identified even if the user renames the file), and does not call "s4384473088-d968438914" the name.

Reply:
Mmh ok, I'm not strongly against but I think it is a bit smell code to have the duplicate. I even envisage to store s = "4384473088" and s = "968438914" separatly so we can index them and quickly build  elo graph over "d" (does that makes sense ?)